src/components/Button.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../styles/Button.css';
+
+const Button = ({ buttonName, color, wide }) => (
+  <button
+    className="button"
+    type="button"
+    style={{
+      backgroundColor: color,
+      width: wide ? '50%' : null,
```

Comment:
Why are you using `null` when `wide` is `false`? 🤔 
I think you should use `25%` instead of `null`.

> - buttons take 25% of container width (except `0` button) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/25Wide.png))
> - `wide` is boolean, true means that button take 50% of space (used for `0` button) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/WideProps.png))

Reply:
I was first initializing the width through the CSS file. But now I have updated it.